docs/python/settings-reference.md

```diff
@@ -68,6 +69,8 @@ The language server settings apply when `python.languageServer` is `Pylance` or
 | inlayHints.functionReturnTypes | false |  Whether to display inlay hints for function return types.  Accepted values are `true` or `false`. |
 | inlayHints.pytestParameters | false | Whether to display inlay hints for pytest fixture argument types. Accepted values are `true` or `false`. |
 | diagnosticSeverityOverrides | {} | Allows a user to override the severity levels for individual diagnostics. <br> For each rule, the available severity levels are `error` (red squiggle), `warning` (yellow squiggle), `information` (blue squiggle), and `none` (rule disabled). <br> For information about the keys to use for the diagnostic severity rules, see the **Diagnostic severity rules** section below. |
+| fixAll | `[]` | A list of code actions to run when running the **Fix All** command. <br> Accepted values in this list are `source.unusedImports` to remove all unused imports in the open file, and `source.convertImportFormat` to convert the imports according to the `python.analysis.importFormat` setting. |
```

Comment:
cc @heejaechang @rchiodo @debonte @bschnurr to make sure this sounds right :)

Reply:
`fix all` can be issued from "fix all" command

![image](https://user-images.githubusercontent.com/1333179/222869811-f7754a9e-e68b-4ccb-ac44-03fb50225f74.png)

or "editor.codeActionsOnSave"

![image](https://user-images.githubusercontent.com/1333179/222869861-4fa75402-ddb0-4c5f-a666-433f50999ec0.png)

and yes, currently we only support 2 source actions you listed above. but the list can grow. so might be better to leave the description as open ends?

we have [README](https://github.com/microsoft/pyrx/blob/main/packages/vscode-pylance/README.md) that lists accepted value

![image](https://user-images.githubusercontent.com/1333179/222870016-844fc330-6f71-4a46-bf03-46c1bedd5722.png)